app/controllers/users_controller.rb

```diff
@@ -0,0 +1,19 @@
+class UsersController < ApplicationController
+  def new
+  end
+
+  def create
+    @user = User.create(param_method) 
```

Comment:
This will already create the user record this what's the need to call `@user.save`?